### DISCUSSION TITLE
Default midi mapping for currently selected clip #enhancement

### DISCUSSION THREAD

**Discussion by kleinstar1** [SynthstromAudible/DelugeFirmware#260] [Ideas]
Hi,

I am planning an implementation of a default midi mapping which applies on the current selected clip.
Similar as things work on elektron devices.

It could work like this:
Setting up one Midi channel in settings (..Default midi channel ..auto channel) - on this channel a Midi table applies just for the selected clip or kit. This could include Notes for kit rows by number to play drums with pads and notes play the selected synth clip.

Due to the multiple clip architecture of the deluge - I guess, that this is the only way of implementing a predefined midi mapping.
This would be a very big deal for everyone I guess: We could map a controller, go in a clip and instantly change all parameters.
I find, that I am really lazy with changing shortcut-parametes.

I think this will take me forever to implement - I am C# guy and dont have jlink or so - but love the deluge like we all :)
And don`t know how to set the label #enhancement :) 
What do you think?

**Comment by Maly-Lemire** [comment]
Also a C# guy here but would love that feature!
